src/models/activityReportCollaborator.js

```diff
@@ -16,6 +33,23 @@ module.exports = (sequelize, DataTypes) => {
       allowNull: false,
       type: DataTypes.INTEGER,
     },
+    fullName: {
+      type: DataTypes.VIRTUAL,
+      get() {
+        return this.collaboratorRoles
+          ? generateFullName(this.user, this.collaboratorRoles)
+          : this.user.name;
+      },
+    },
+    fullNameSubstituteRoles: {
```

Comment:
do we need two functions here or can we just check the roles.length in ```generateFullName```?

Reply:
Yea this was sort of a place holder.

I'm not sure if the reports in alerts should show missing roles.